src/js/login.js

```diff
@@ -44,6 +44,50 @@ login.prototype.getEnvDetail = function() {
     return false;
 }
 
+/*
+ * Get cell URL from URL
+ * Parameter:
+ *     "https://demo.personium.io/debug-user1/test/login.html"
+ * Return:
+ *     "https://demo.personium.io/debug-user1/"
+ */
+login.prototype.cellUrlWithEndingSlash = function (tempUrl) {
```

Comment:
Please rewrite the method in the following way.  

        login.prototype.getCellInfo = function (cellUrlSplit) {
            let rootURL = _.first(cellUrlSplit, 3).join("/") + "/";
            let cellURL = _.first(cellUrlSplit, 4).join("/") + "/";
            let cellName = this.getName(cellURL);

            return {
                rootURL: rootURL,
                cellURL: cellURL,
                cellName: cellName
            };
        }